gems/aws-sdk-core/spec/aws/waiters_spec.rb

```diff
@@ -292,6 +376,31 @@ module Waiters
               expect(retries).to be(1)
             end
 
+            it 'can match array of objects' do
+              client.stub_responses(:waiter_operation,
+                                    table_list: [{ table_status: 'ACTIVE' }, { table_status: 'ACTIVE' }])
```

Comment:
Could fix the alignment like so - I think rubocop will ok with the below:

```suggestion
              client.stub_responses(
                :waiter_operation,
                table_list: [ { table_status: 'ACTIVE' }, { table_status: 'ACTIVE' }]
              )
```